source/features/latest-tag-button.tsx

```diff
@@ -2,19 +2,21 @@ import './latest-tag-button.css';
 import React from 'dom-chef';
 import cache from 'webext-storage-cache';
 import TagIcon from 'octicon/tag.svg';
+import DiffIcon from 'octicon/diff.svg';
 import elementReady from 'element-ready';
 import * as pageDetect from 'github-url-detection';
 
 import features from '.';
 import * as api from '../github-helpers/api';
-import fetchDom from '../helpers/fetch-dom';
+import pluralize from '../helpers/pluralize';
 import GitHubURL from '../github-helpers/github-url';
+import {groupButtons} from '../github-helpers/group-buttons';
 import getDefaultBranch from '../github-helpers/get-default-branch';
 import {getRepoURL, getCurrentBranch, getRepoGQL, getLatestVersionTag} from '../github-helpers';
 
 interface RepoPublishState {
 	latestTag: string | false;
-	isUpToDate: boolean;
+	aheadBy?: number | undefined;
```

Comment:
Is that required?

```suggestion
	aheadBy?: number;
```

My other suggestions below should make this code work.